lib/auth.coffee

```diff
@@ -274,6 +274,46 @@ getAuth = (deps, opts) ->
 		.get('id')
 		.asCallback(callback)
 
+	###*
+	# @summary Get current logged in user's organization id
+	# @name getUserOrganizationId
+	# @private
+	# @function
+	# @memberof resin.auth
+	#
+	# @description This will only work if you used {@link module:resin.auth.login} to log in.
+	#
+	# @fulfil {Number} - user's organization id
+	# @returns {Promise}
+	#
+	# @example
+	# resin.auth.getUserOrganizationId().then(function(organizationId) {
+	# 	console.log(organizationId);
+	# });
+	#
+	# @example
+	# resin.auth.getUserOrganizationId(function(error, organizationId) {
+	# 	if (error) throw error;
+	# 	console.log(organizationId);
+	# });
+	###
+	exports.getUserOrganizationId = (callback) ->
```

Comment:
Can we call these 'personal' organizations? Feels a little clearer to me, and I think that's what we've done in a few other places (though do correct me if that's not true!). Life will be simpler if we come up with some consistent terms early on :-).